README.md

```diff
@@ -73,7 +73,6 @@ Available variables include the following:
 | Variable | Default | Purpose |
 |----------|---------|---------|
 | `DABL_HEALTH_PORT` | 8089 | Port for Health/Status HTTP endpoint |
-| `DABL_INTEGRATION_INTEGRATION_ID` | | Integration ID (arbitrary for testing, but required) |
```

Comment:
Turns out we didn't need this at all within an integration at runtime.

Reply:
nice